README.md

```diff
@@ -29,6 +29,14 @@ ember install ember-frost-sort
 | `sortParams` | `array` | `[{"direction: "asc/desc", "value": <attr-name>}]` |  Array that specifies the sort order. |
 | `onChange` | `string` | `<action-name>` | The action to call when the value of the select item changes. |
 
+## Testing with ember-hook
+The sort component is accessible using ember-hook with the top level hook name or you can access the internal components as well -
+* Add filter button hook - `$hook('<hook-name>-add-filter')`
```

Comment:
I deliberated on the naming and went back and forth. I am thinking that at the top level where this is used, that the name most likely to be concatenated is `sort` which results in the hook `list-sort-add-filter` if it is being used in a list for example. Perhaps, using the previous example, `list-sort-add-sort` would make more sense but I was trying to avoid the repetition of the word `sort`.